tests/testcases/base_test.py

```diff
@@ -20,7 +20,13 @@ def __init__(self, *args, **kwargs):
 
     @classmethod
     def setUpClass(cls):
-        pass
+        cls.vms = []
```

Comment:
I prefer to do that in setUp and tearDown Methods, If we have an issue in deleting a specific `primitives`, We will knw which test cases cause that.

Reply:
???

Reply:
i did it, but in tearDown and this var still as it is